ansible/roles/hoover/templates/docker/docker-compose.yml

```diff
@@ -0,0 +1,58 @@
+version: "2"
```

Comment:
why is a version: "2" in a new file?

Reply:
It's the docker-compose file format. Version 3 is current, but we're using old versions of docker and docker-compose (from xenial), so we're at version 2.